plugins/by-name/aw-watcher/default.nix

```diff
@@ -0,0 +1,40 @@
+{ lib, ... }:
+let
+  inherit (lib.nixvim) defaultNullOpts;
+in
+lib.nixvim.plugins.mkNeovimPlugin {
+  name = "aw-watcher";
+  moduleName = "aw_watcher";
+  packPathName = "aw_watcher";
```

Comment:
```suggestion
  packPathName = "aw-watcher.nvim";
```

Reply:
I insist, by `packPathName`, we understand the "project name" or official name for this plugin.
In this case, it is `aw-watcher.nvim`. This value is only used for display purposes.

Reply:
This isn't _only_ used for display purposes. It is also used for lazyloading.